lib/appsignal/check_in/cron.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module CheckIn
+    class Cron
+      class << self
+        # @api private
+        def transmitter
+          @transmitter ||= Appsignal::Transmitter.new(
+            "#{Appsignal.config[:logging_endpoint]}/check_ins/json"
+          )
+        end
+      end
+
+      attr_reader :name, :id
+
+      def initialize(name:)
+        @name = name
```

Comment:
In the new setup the domain terms would be: name -> Identifier, id -> digest. Not sure if we want to use that here as well?

Reply:
We can also make it a positional argument if we don't expect any more arguments. Then we don't have to worry about naming things.

Reply:
Thanks for pointing this out, I'll see about amending this across integrations.

Reply:
This is a bit trickier than it sounds because the same objects need to conform to the previous and new attribute and accessor names. Might have to wait until Monday.